spec/features/likes_before_cool_features_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.feature "Likes on hipstergrams" do
+ 
+  before { 
+    sign_in(sign_up)
+    add_picture            
+  }
+
+
+
+ it 'a user can like an hipstergram, before it gets cool.', js: true do
+  visit '/hipstergrams'
+  click_link 'Like before it is cool'
+  expect(page).to have_content("1 Like")
+end
```

Comment:
Lint/BlockAlignment: `end` at 16, 0 is not aligned with `it 'a user can like an hipstergram, before it gets cool.', js: true do` at 12, 1.